packages/indexer/src/plugins/kv.ts

```diff
@@ -1,128 +1,130 @@
 import assert from "node:assert";
 import type { Cursor, DataFinality } from "@apibara/protocol";
-import { type Database, type ISqlite, open } from "sqlite";
+import type { Database as SqliteDatabase, Statement } from "better-sqlite3";
 import { useIndexerContext } from "../context";
 import { deserialize, serialize } from "../vcr";
 import { defineIndexerPlugin } from "./config";
 
-type SqliteArgs = ISqlite.Config;
-
-export function kv<TFilter, TBlock, TRet>(args: SqliteArgs) {
+export function kv<TFilter, TBlock, TRet>(db: SqliteDatabase) {
```

Comment:
I think it's better to have named parameters so if in the future we add any option we don't need to make a breaking change.

```suggestion
export function kv<TFilter, TBlock, TRet>({ database }: { database: SqliteDatabase }) {
```

Reply:
Same for all other instances.